variables.tf

```diff
@@ -0,0 +1,35 @@
+variable "create_config_authorization" {
+  description = "Toggle that controls creation/management of a config authorization"
+  type        = "string"
```

Comment:
I think `type = "string"` should be removed as well.

Reply:
Fixed